### ISSUE TITLE
Added printing speed in Bambu Studio.

### ISSUE THREAD

**Issue by PHIL-IP** [bambulab/BambuStudio#1417]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Addition of a menu allowing to launch a printing directly in silent mode or sport etc...
At the moment you have to start printing first and then change the value of the printing speed.

**Describe alternatives you've considered**
None except to start the printing and then go to change the speed of it

**Comment by Somerandomgithugguy** [comment]
Yes we really need this, it is so annoying to have to manually change it every time after starting the print!

**Comment by Somerandomgithugguy** [comment]
Same issue as raised here #1290  and #1419

**Comment by daliudzius** [comment]
This feature request would be to slice a project with easy speed settings included - for example slicing with "silent" speed already set in Bambu Studio so we don't have to wait for the print to start to toggle speed settings. 

Both issues [#1290](https://github.com/bambulab/BambuStudio/issues/1290) and [#1419](https://github.com/bambulab/BambuStudio/issues/1419) relate only to adding additional speed options.

**Comment by aliasmeum** [comment]
Before version 1.7 it was possible to change to silent mode as soon as the slicer jumped to the device page. Now you have to wait (I believe, until the bed is heated?). If you try to change the speed before this, it just jumps back to 100%. Having to sit around waiting for the bed to heat to be able to change the speed is ridiculously annoying. Could ability to choose speed in the "Send to printer" window be an option? Preferably by setting the percentage.

**Comment by Somerandomgithugguy** [comment]
It is so sad that this is not implemented yet. :(

**Comment by anoblet** [comment]
Is there any update on this?

**Comment by Somerandomgithugguy** [comment]
> Is there any update on this?

Sadly not...

**Comment by sodachen22** [comment]
We got this feature request and will evaluate it. Thanks you!

**Comment by kzlsnn** [comment]
It would be great if this feature was added thx.

**Comment by MerlinStWahgwaan** [comment]
Would love this feature!!! 
Been trying for 2 hours to accomplish this today, only to find out its not possible in Prepare to set Silent mode, or even to just adjust the speed easily, at all...
Like, I dont need 50% speed reduction, but going to 80 or 75% would very much improve my prints, and I cant do that either.
A simple text option in the Global Speed settings to input a speed ratio manually cant be that difficult. 
50, 80, 100, 120, 150, what ever percentage upto what ever Ludicrous speed is compared to Standard.
Even better would be if there was a setting within the individual Plate settings, and even per object and before/after specific layer options.

The more control, the better print quality anyone can achieve with any materials

**Comment by satoanderson** [comment]
+1 to that
